frontend/src/metabase/css/components/header.css

```diff
@@ -54,26 +54,34 @@
 .EditHeader .Button {
     color: var(--brand-color);
     border: none;
-    text-transform: uppercase;
-    font-size: 0.75rem;
-    background-color: rgba(255,255,255,0.5);
-    font-weight: normal;
+    font-size: 0.875rem;
+    background-color: rgba(255,255,255,0.1);
+    font-weight: 700;
     margin-left: 0.75em;
 }
 
 .EditHeader .Button--primary {
     background-color: white;
+    color: #444444;
```

Comment:
Since we're using this value in two places here I'd consider maybe adding it as a var?